leetcode/src/2024.c

```diff
@@ -0,0 +1,33 @@
+#define max(X, Y) ((X) > (Y) ? (X) : (Y))
+
+int getMaxLength(char * answerKey, char targetChar, int k){

```

Comment:
```suggestion
int maximizeTarget(char * answerKey, char targetChar, int k){
```

I believe you're trying to maximise the number of `'T'` or `'F'` not "getting" the max length.